src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -287,14 +288,35 @@ public override bool TryMapToProjectedDocumentRange(RazorCodeDocument codeDocume
                 return false;
             }
 
-            projectedRange = new Range(
+            projectedRange = new Omni.Range(
                 projectedStart,
                 projectedEnd);
 
             return true;
         }
 
-        public override bool TryMapFromProjectedDocumentPosition(RazorCodeDocument codeDocument, int csharpAbsoluteIndex, [NotNullWhen(true)] out Position? originalPosition, out int originalIndex)
+        public override bool TryMapToProjectedDocumentVSRange(RazorCodeDocument razorCodeDocument, Range range, [NotNullWhen(true)] out Range? projectedRange)
```

Comment:
Love the VS  Suffix approach you took as an interim till we can fully get rid of O#